pkgs/by-name/sv/svox/package.nix

```
@@ -8,16 +8,20 @@
 
 stdenv.mkDerivation {
   pname = "svox";
-  version = "2018-02-14";
+  version = "2021-05-06";
```

The current policy is to use `${lastVersion}-unstable-${commitDate}` for commits:

```suggestion
  version = "0-unstable-2021-05-06";
```